src/udev/udev-builtin-net_id.c

```diff
@@ -897,6 +938,20 @@ static int builtin_net_id(sd_device *dev, int argc, char *argv[], bool test) {
                 return 0;
         }
 
+        /* get netdevsim path names */
+        if (names_netdevsim(dev, &names) >= 0 && names.type == NET_NETDEVSIM) {
+                char str[IFNAMSIZ];
+
+                if (names.netdevsim_path[0] &&
```

Comment:
Why check `names.netdevsim_path[0]`? `names_netdevsim` does all the checks already.